apps/monitor-client/lib/app/AppContainer.tsx

```diff
@@ -1,54 +1,12 @@
-import React, { useContext, useState, useEffect } from 'react'
-import { AppContext, IAppContext } from '../context';
+import React from 'react';
+
 import { Login } from '../features/auth/Login';
+import { HomePage } from '../features/HomePage';
 
 export function AppContainer() {
```

Comment:
the component should contain high level routing

Reply:
done, but route "/login" wont work